content/configuration/output.md

```diff
@@ -312,7 +312,46 @@ T> Read the [authoring libraries guide](/guides/author-libraries) guide for more
 
 `string` or `string[]` (since webpack 3.0.0)
 
-Allows to select an export for the library.
+> Default: `_entry_return_`
```

Comment:
Something like "The default value `_entry_return_` is the namespace or default module returned by your entry file." ...wording?